addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -155,7 +154,7 @@ def portal_my_opportunities(self, page=1, date_begin=None, date_end=None, sortby
             'pager': pager,
             'searchbar_sortings': searchbar_sortings,
             'sortby': sortby,
-            'searchbar_filters': OrderedDict(sorted(searchbar_filters.items())),
+            'searchbar_filters': OrderedDict(searchbar_filters.items()),
```

Comment:
I think keys ordering is guaranteed in our python version, so we could simplify and just pass the dict.